### PR TITLE
Supplementary documents

### PR DESCRIPTION
**Pull request by zkh2016**
### PR types

Others
### PR changes

Docs
### Describe

补充values，to_dense和to_sparse_coo的英文文档。对应代码[PR](https://github.com/PaddlePaddle/Paddle/pull/41394)

### CODE REVIEW

File: python/paddle/fluid/dygraph/varbase_patch_methods.py [hunk 1]
```diff
@@ -893,6 +933,25 @@ def to_dense(self):
 
     @framework.dygraph_only
     def to_sparse_coo(self, sparse_dim):
+        """
+        **Notes**:
+            **This API is ONLY available in Dygraph mode**
+        Convert the current DenseTensor to SparseTensor in COO format.
+        Returns:
```
Comment (TCChenlong) [inline python/paddle/fluid/dygraph/varbase_patch_methods.py:947]: Add a blank line before ``Returns``.

File: python/paddle/fluid/dygraph/varbase_patch_methods.py [hunk 2]
```diff
@@ -893,6 +933,25 @@ def to_dense(self):
...
+            Tensor: A SparseCooTensor
+        Examples:
```
Comment (TCChenlong) [inline python/paddle/fluid/dygraph/varbase_patch_methods.py:950]: Add a blank line before ``Examples``.

File: python/paddle/fluid/dygraph/varbase_patch_methods.py [hunk 3]
```diff
@@ -893,6 +933,25 @@ def to_dense(self):
...
+            .. code-block:: python
+                import paddle
```
Comment (TCChenlong) [inline python/paddle/fluid/dygraph/varbase_patch_methods.py:953]: Add a blank line before ``import paddle``.

File: python/paddle/fluid/dygraph/varbase_patch_methods.py (hunk 1 above)
Comment (zkh2016) [reply python/paddle/fluid/dygraph/varbase_patch_methods.py:947]: done

File: python/paddle/fluid/dygraph/varbase_patch_methods.py (hunk 2 above)
Comment (zkh2016) [reply python/paddle/fluid/dygraph/varbase_patch_methods.py:950]: done

File: python/paddle/fluid/dygraph/varbase_patch_methods.py (hunk 3 above)
Comment (zkh2016) [reply python/paddle/fluid/dygraph/varbase_patch_methods.py:953]: done

**Review by TCChenlong — APPROVED** [review]

**Pull request merged**